app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,9 @@
   </head>
 
   <body>
+    <% flash.each do |type, message| %>
+      <%= content_tag :div, message, class: type %>
```

Comment:
Super curious about whether there are other text_helper methods that would be helpful for us. @CjMoore i'm going to add a card to remind myself to do some research in the docs if that's ok.